src/averageAges.js

```diff
@@ -15,6 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const man = people.filter(person => person.sex === 'm');
+  const men = century
+    ? man.filter(
+      (person) => Math.ceil(person.died / 100) === century
+    )
+    : man;
```

Comment:
```suggestion
```
I Think here its better to use a simple 'if' statement, what filters the original woman array when some parameter is valid, if we want to add more params in the future, its easier to do with this method. U can use the same logic further.